200-CNAB-registries.md

```diff
@@ -15,3 +16,39 @@ This specification describes how CNAB bundles can be stored inside of OCI Regist
 
 > Previous versions of the CNAB Core draft specification included a repository protocol. That work is completely superseded by this specification.
+
+This specification is distinct from the CNAB Core specification. An implementation may comply with the CNAB Core specification, and yet not comply with this specification.
+
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt). The use of these keywords in this document are statements about how a CNAB implementation may fulfill the CNAB registry storage specification only.
+
+The CNAB Core 1.0 specification (CNAB1) does not dictate how bundles should be distributed. This is intentional, so that organizations that already have a way of distributing artifacts may continue to use it.
+This document, the CNAB Registries specification, is not part of the core specification, and an implementation of CNAB Core MAY NOT implement it, yet still claim compliance with CNAB Core. A CNAB implementation MUST NOT claim to be CNAB Registries compliant unless it meets this specification.
+
+This specification proposes the use of [OCI registries][oci-org] to distribute CNAB artifacts.
+
+## Approach
+
+<!--
+> xxx intention is to reuse existing container infrastructure for sharing of CNABs
+xxx CNAB is a collection of objects (bundle.json, images) making OCI index a natural fit
+xxx OCI index for bundles which store all components in the same repo (thin bundles could just be the bundle.json at index[0] with no other images)
+xxx want this to work with signing (section 3xx)
+xxx want to keep bundle.json the same when moving between repos/registries (signing)
+xxx reference implementation: https://github.com/cnabio/cnab-to-oci
+-->
+
+Container registries provide a reliable and highly scalable distribution for container images, and implementations of container registries can be found in cloud provider services, vendor services, or open source projects, and more and more artifact types are distributed with OCI registries, process standardized by the [OCI Artifacts][artifacts] project.
+
+A Cloud Native Application Bundle is a collection of metadata and container images that are needed in order to successfully deploy and application - as such, it is not a _single_ new artifact, but represents a _collection of multiple artifacts_.
+
+An [OCI image index (or simply OCI index)][oci-index] represents a collection of container images stored in a repository - so rather than using a new artifact, this specification proposes the use of an OCI index to represent Cloud Native Application Bundles, and use it as a distribution mechanism for the bundle file, invocation image, and images used by the application.
```

Comment:
```suggestion
An [OCI image index (or simply OCI index)][oci-index] represents a collection of container images stored in a repository - so rather than using a new artifact, this specification proposes reusing the OCI index to represent Cloud Native Application Bundles, and use it as a natural distribution mechanism for the bundle file, invocation image, and images used by the application.
```